src/renderer/pages/Staking/Operations/components/OperationForm/OperationForm.tsx

```diff
@@ -80,13 +80,16 @@ export const OperationForm = ({
 }: Props) => {
   const { t } = useI18n();
   const dbAccounts = useUnit(walletModel.$accounts);
+  const dbWallets = useUnit(walletModel.$wallets);
 
   const destinations = getDestinations(t);
 
   const [activePayout, setActivePayout] = useState<Address>('');
   const [payoutAccounts, setPayoutAccounts] = useState<ComboboxOption<Address>[]>([]);
 
-  const destAccounts = dbAccounts.filter((a) => accountUtils.isChainIdMatch(a, chainId));
+  const destAccounts = filterPvRootAccounts(dbAccounts, dbWallets).filter((a) =>
```

Comment:
Let's create a whole function to get dest accounts (include `filterPvRootAccounts` logic into `getDestinationAccounts(wallets, accounts, chainId)`)